pandas/tests/extension_arrays/test_common.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
```

Comment:
put this in ``pandas/tests/extension/test_common.py`` you also need an ``__init__.py``